update.sh

```diff
@@ -1,218 +1,269 @@
 #!/bin/bash
 
+
+hash jq 2>/dev/null || { printf >&2 "'jq' required, but not found.\nInstall via:\n  sudo apt install jq\nAborting.\n"; exit 1; }
+hash yq 2>/dev/null || { printf >&2 "'yq' required, but not found.\nInstall via:\n  python3 -m pip install yq\nAborting.\n"; exit 1; }
+hash docker compose 2>/dev/null || { printf >&2 "'docker compose' required, but not found.\nInstall via: https://docs.docker.com/compose/install/\nAborting.\n"; exit 1; }
```

Comment:
This doesn't work (at least on MacOS):
```
> hash docker compose
hash: no such command: compose
```

Reply:
`hash docker compose` should through an error, that line is to detect it not being installed and abort the script.

Running without `docker compose` installed should result in
```
'docker compose' required, but not found.
Install via: https://docs.docker.com/compose/install/
Aborting.
```